common/src/View/MoveBrushEdgesCommand.cpp

```diff
@@ -62,9 +62,15 @@ namespace TrenchBroom {
         }
 
         bool MoveBrushEdgesCommand::doCollateWith(UndoableCommand::Ptr command) {
-            // Don't collate vertex moves. Collation changes the path along which the vertices are moved, and as a result
-            // changes the outcome of the entire operation.
-            return false;
+            MoveBrushEdgesCommand* other = static_cast<MoveBrushEdgesCommand*>(command.get());
+
+            if (!VectorUtils::equals(m_newEdgePositions, other->m_oldEdgePositions))
```

Comment:
I think all that matters here is whether `this's` snapshot will undo all the changes in `other`.  i.e. if other's snapshot contains any brushes not in this's snapshot, return false, otherwise they can be collated.

The current code looks fine though so maybe it's not worth changing at this point.